WebDevStudios/nextjs-wordpress-starter
# Feature/9 Yoast Seo

Closes #9

### DESCRIPTION ###
This PR exposes `globalSeo` which contains the Yoast SEO global settings in `Index` and `Post` component. This also exposes `seo` in `Meta` component which contains single post Yoast SEO settings.

This PR also fixes the issue where `<meta property="og:image" />` is render twice in `[slug].js` due to being called in both `Post` and `Meta` components.

### SCREENSHOTS ###
![meta](https://user-images.githubusercontent.com/5747475/94691117-800e5280-0363-11eb-9810-695e3070e824.jpg)

### STEPS TO VERIFY ###
* Global Yoast SEO settings changes in https://nextjs.wpengine.com/ should be visible in the `globalSeo` object. You're gonna need to console log this in the code as we are not rendering the Global Settings anywhere for now.
* As for individual post SEO, you can go to a single post https://nextjs.wpengine.com/wp-admin/post.php?post=12&action=edit then edit the Meta description
![yoast-seo](https://user-images.githubusercontent.com/5747475/94692346-05decd80-0365-11eb-8de6-b6b0e5510a84.jpg)

Then you should be able to see it as `<meta name="description" />` in the nextjs.
